index.html

```diff
@@ -1,13 +1,36 @@
 <!doctype html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<head>
+    <meta charset="UTF-8"/>
+    <link rel="icon" type="image/svg+xml" href="/vite.svg"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <title>Vite App</title>
-  </head>
-  <body>
-    <div id="app"></div>
-    <script type="module" src="/main.js"></script>
-  </body>
+</head>
+<body>
+<div id="app">
+<!--    <div class="hover-text">hover me-->
+<!--        <span class="tooltip-text" id="top">I'm a tooltip!</span>-->
+<!--    </div>-->
+    <button class="tooltip-hover" id="tooltip-hover-button">Button<span class="tooltip-text" id="tooltip-hover">Example</span></button>
+    <button class="tooltip-hover" id="tooltip-button">Button<span class="tooltip-text" id="tooltip">Example</span></button>
```

Comment:
You can try do the trick and make an data-attribute like so:
```html
<button type="button" data-tooltip="Example">Button</button>
```

And create tooltip on fly using this data attribute.
P.s.: Don't forget to add `type="button"` for buttons, because it has default behaviour as "submit"